Core/src/org/sleuthkit/autopsy/casemodule/services/TagOptionsPanel.java

```diff
@@ -190,7 +191,7 @@ private void newTagNameButtonActionPerformed(java.awt.event.ActionEvent evt) {//
         NewTagNameDialog.BUTTON_PRESSED result = dialog.getResult();
         if (result == NewTagNameDialog.BUTTON_PRESSED.OK) {
             String newTagDisplayName = dialog.getTagName();
-            TagNameDefiniton newTagType = new TagNameDefiniton(newTagDisplayName, DEFAULT_DESCRIPTION, DEFAULT_COLOR);
+            TagNameDefiniton newTagType = new TagNameDefiniton(newTagDisplayName, DEFAULT_DESCRIPTION, DEFAULT_COLOR, TskData.FileKnown.UNKNOWN);
```

Comment:
Was the old code here using place holders for non-editable properties? Is editing not included in this PR?